docs/sdks/sdk-features/obfuscation.md

```diff
@@ -45,6 +45,8 @@ The `value` field in `conditions` may or may not be hashed depending on the conf
 | greater than (>)           | encoded |
 | greater than or equal (>=) | encoded |
 | matches regex              | encoded |
+| is null                    | n/a     |
+| is not null                | n/a     |
```

Comment:
Nit: in the SDK this is a single operator where the value indicates true or false, which is hashed